### ISSUE TITLE
A confusion about mrc model

### ISSUE THREAD

**Issue by Maybewuss** [huggingface/transformers#7260]
# ❓ Questions & Help

- I found all of pretrain models using for mrc use softmax to pred start and end probability. 

- If a paragraph has multi-answers, softmax not worked, otherwise, adding some postprocess to combine start and end probability to sort that we can choose n-best answer, but the numbert is  fixed for all paragaph.
- Squad 2.0 has unanswerable questions and i found  data processor define start index == end index == 0 to handle this situation.
- So my confusion is why not use sigmoid instead of softmax in the last layer, if i use sigmoid, i can deal with muti-answers or non-answer easily....
